src/components/CardContainer/CardContainer.js

```diff
@@ -30,12 +30,28 @@ class CardContainer extends React.Component {
             .then(res => {
                 let random = UniqueRandomArray(res)
                 let dogs = [random(), random()]
-                const state = {...this.state, dogs: dogs}
+                const state = {...this.state, dogs: dogs, randomDog: random}
                 this.setState(state)
                 }
             )
     }
 
+    clickUpdateWin = (id) => {
+        fetch('http://localhost:3000/dogs/' + id + "/win", {
```

Comment:
We've used this URL string a few times on the class, so to reduce duplication perhaps we could set this on the class or file as immutable somehow

Reply:
on it